jetstream_pt/ray_worker.py

```diff
@@ -490,7 +487,7 @@ def prefill_ray(
         samples_per_slot=1,
     )
 
-    return prefix, result
+    return result
```

Comment:
Let's keep the same function signature between interleave and disaggregated serving. Return None, result instead of return result.